exonum-java-binding/core/rust/src/runtime/service_factory_adapter.rs

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 The Exonum Team
```

Comment:
The name of this file is actually conflicting with the name from #776. (it contains the same kind of refactoring)

Reply:
Is it a problem? Perhaps it is, especially for one who's merging last ))